rules/azure-naming-resources/rule.md

```diff
@@ -36,8 +36,8 @@ Organizing your cloud assets starts with good names. It is best to be consistent
 
 * All lower case 
 * Use [kebab case (“-“ as a separator)](https://www.ssw.com.au/rules/use-dashes-in-urls)
+* Include an [resource type abbreviation](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations) (so it's easy to find the resource in a script)
```

Comment:
Typo "an".
Change to "Include **a** resource type abbreviation"

Reply:
done, next time can you just use the suggested change button